src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit add']//*[text()='Add Product']")
+	private WebElement clickAddProductButton;
+
+	/**
+	 * Method to click on Add Product Button".
+	 * 
+	 * @throws IOException
+	 */
+	public void clickAddProductButton() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Click on Add Product Button");
+		Assert.assertTrue(clickAddProductButton.isDisplayed());
+		clickAddProductButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='product.descriptions[0].content']")
+	private WebElement enterEnglishDescription;
+
+	/**
+	 * Method to Enter English Description.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterEnglishDescription() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter English Description");
+		Assert.assertTrue(enterEnglishDescription.isDisplayed());
+		enterEnglishDescription.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 3));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='product.number']")
+	private WebElement enterProductCode;
+
+	/**
+	 * Method to Enter Product Code.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterProductCode() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Product Code");
+		Assert.assertTrue(enterProductCode.isDisplayed());
+		enterProductCode.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 3));
+
+	}
+
+	public void selectCompany() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		WebElement Companieselement = driver.findElement(By.xpath("//select[@name='product.entities']"));
+		Select se = new Select(Companieselement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 2, 0));
+
+	}
+	
+	private WebElement selectPriceCurrency;
+
+	/**
+	 * Method to select price currency.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectPriceCurrency() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		WebElement Currencyelement = driver.findElement(By.xpath("//select[@name='product.currencies']"));
+		Select se = new Select(Currencyelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='product.rate']")
+	private WebElement enterPriceRate;
+
+	/**
+	 * Method to Enter Price Rate.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterPriceRate() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Price Rate");
+		JavaScriptExec.sleep();
+		WaitClass.WaitForElementisDisplay(driver, 10, enterPriceRate);
+		Assert.assertTrue(enterPriceRate.isDisplayed());
+		//enterPriceRate1.clear();
+		enterPriceRate.sendKeys(BasePage.getCellData(xlsxName, sheetName, 6, 3));
```

Comment:
@mayankchhajer remove this commented line

Reply:
Done